test/C/src/arduino/BlinkAttemptThreading.lf

```diff
@@ -8,7 +8,7 @@ target C {
     name: "arduino",
     board: "arduino:avr:mega"
   },
-  threading: true
+  single-threaded: false
```

Comment:
Should this instead be a `workers` property?

Reply:
I think we can just remove the property also since threading is the default option?

Reply:
True, I meant to preserve the original logic but in this case the `single-threaded: false` is just redundant. I'll remove the property.